src/Utilities/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataSymbolMap.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.FlowAnalysis.DataFlow;
+
+namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
+{
+    /// <summary>
+    /// Mapping of <see cref="ITypeSymbol"/> to <see cref="ITaintedDataInfo"/> (tainted data source/sanitizer/sink info).
+    /// </summary>
+#pragma warning disable CA1812   // TaintedDataSymbolMap is used in other projects.
```

Comment:
>CA1812 [](start = 24, length = 6)

Can you file a bug for this? We should fix CA1812 to not fire here..

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/1893